foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/store/Property.java

```diff
@@ -49,9 +49,9 @@ public void setValue(String value) {
     public String toString() {
         return "Property{" +
                 "name='" + name +
-                ", value='" + value +
-                ", groupName='" + groupName +
-                '}';
+                "', value='" + value +
```

Comment:
S: I know it was already there and we kind of touched on this in a different PR, but I wonder if we should just remove `value` from toString all together just to really prevent in silly mistakes.